src/Core/Package.vala

```diff
@@ -341,6 +336,11 @@ public class AppCenterCore.Package : Object {
                     return _("Ubuntu (non-curated)");
                 }
             } else if (backend is FlatpakBackend) {
+                var fp_package = this as FlatpakPackage;
+                if (fp_package != null && fp_package.installation == FlatpakBackend.system_installation) {
+                    return _("%s Systemwide (non-curated)").printf (origin);
```

Comment:
Hrm, I don't know the best way to communicate this inside a combobox. I think at least hyphenated (`system-wide`) or just `system`? And maybe since we automatically select the "right" source and have non-curated warnings anyway, we could get rid of the `(non-curated)` bit here (and below, to match) to clean things up?

```suggestion
                    return _("%s (system-wide)").printf (origin);
```